ros2_foxglove_bridge/src/ros2_foxglove_bridge.cpp

```diff
@@ -582,7 +582,9 @@ void FoxgloveBridge::clientAdvertise(const foxglove::ClientAdvertisement& advert
                    });
     const rclcpp::QoS qos = otherPublisherIt == otherPublishers.end()
                               ? rclcpp::SystemDefaultsQoS()
-                              : otherPublisherIt->qos_profile();
+                              : rclcpp::QoS(otherPublisherIt->qos_profile())
+                                  // Override the history policy as it can be `unknown`
```

Comment:
Would it be better to only override _if_ it is `unknown` and leave it otherwise?

Reply:
Yes, will do. Although it seems that this is ROS distro specific. On `humble` this is always `unknown` (might be a bug), on  `galactic` the history is always set